### PR TITLE
Ammo hud and updated healthbar

### PR DESCRIPTION
**Pull request by Fueredoriku**

### CODE REVIEW

**Review by toberge — CHANGES_REQUESTED** [review]
Seems to respond correctly to ammo usage, however this is how it looked when I had one shot left in the hatbow:
image
And when depleting chunks of the bar, it sometimes lines up with the gaps between the holes and sometimes not.

**Comment by Fueredoriku** [comment]
>it sometimes lines up with the gaps between the holes and sometimes not.
That is intentional due to varying amounts of ammo used per shot

What I can du though, if need be, is to remove 90 degrees (the last quadrant behind the healthbar) to always show the bar unless empty

**Comment by Fueredoriku** [comment]
The ammoBar when reloading and using the starter body looks a bit weird because the auto-reloads as soon as its magazine is depleted. I want to remove that behaviour in a future PR, so please disregard that specific gun combo looking a bit weird at the moment 😅

File: Assets/Scripts/UI/PlayerHUDController.cs [hunk 1]
```diff
@@ -36,6 +36,7 @@ public class PlayerHUDController : MonoBehaviour
     private Material damageBorder;
 
     private const float ammoSpinDegrees = 30;
+    private const float availableCircumference = 270;
```
Comment (toberge) [inline Assets/Scripts/UI/PlayerHUDController.cs:39]: Why not available _degrees_?

Comment (toberge) [inline Assets/Scripts/UI/PlayerHUDController.cs:39]: Why not available _degrees_?

**Review by toberge — APPROVED** [review]
You fixed the main issue and now the starter body has reload so... Go ahead and merge it (perhaps after considering my renaming suggestion)

File: Assets/Scripts/UI/PlayerHUDController.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/UI/PlayerHUDController.cs:39]: Why is this duplicated smh

Comment (Fueredoriku) [reply Assets/Scripts/UI/PlayerHUDController.cs:39]: Yea, I agree that might be a better name, git push -f incoming

**Pull request merged**